app/modules/academics/schemas.py

```
@@ -1,73 +1,48 @@
     is_current: Optional[bool] = False
     status: Optional[str] = "ACTIVE"
-
 class AcademicYearResponse(AcademicYearCreate):
-    id: int
-    school_id: int
-    created_at: datetime
-    updated_at: Optional[datetime] = None
-
-    class Config:
-        from_attributes = True
+    id: int; school_id: int; created_at: datetime; updated_at: Optional[datetime] = None
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)